lib/styles/trackInputModality.js

```diff
@@ -33,25 +88,39 @@ function setUpEventHandlers(disableFocusRingByDefault) {
 
   document.body.addEventListener(
     'keydown',
-    event => {
-      recentKeyboardEvent = event;
+    () => {
+      hadKeyboardEvent = true;
+      hadClickEvent = false;
 
       if (isHandlingKeyboardThrottle) {
         clearTimeout(isHandlingKeyboardThrottle);
       }
 
       isHandlingKeyboardThrottle = setTimeout(() => {
-        recentKeyboardEvent = null;
+        hadKeyboardEvent = false;
       }, 100);
     },
     true
   );
 
+  document.body.addEventListener('mouseup', () => {
```

Comment:
Was this copied from the previous implementation? Otherwise, would be good to include a comment explaining the rationale for using `mouseup` versus `mousedown` etc.

Reply:
This should be `mousedown` and has been changed. I had been reading some guidance on this, but realized from your comment and revisiting it that I had gotten it backwards. fixed!